parm/params_grib2_tbl_new

```diff
@@ -367,6 +367,7 @@
    0   1   121   0 FSNOWC
    0   6    36   0 FSTRPC
    2   4     5   0 FWINX
+   2   4    26   0 WFIREPOT
```

Comment:
@EricJames-NOAA The file params_grib2_tbl_new should be sorted by grib2 names based on params_grib2_tbl_new.text. Please pick up my params_grib2_tbl_new from /scratch1/NCEPDEV/stmp2/Wen.Meng/eric_james/UPP and commit it to your branch.